libs/@guardian/core-web-vitals/package.json

```diff
@@ -33,7 +33,7 @@
 		"ts-jest": "29.1.1",
 		"tslib": "2.6.2",
 		"typescript": "5.3.3",
-		"web-vitals": "3.5.0",
+		"web-vitals": "4.2.0",
```

Comment:
don't forget to bump the peerdep version too on line 43 (github won't left me suggest it, sorry)

Reply:
thanks @sndrs

Reply:
we need to add a linting rule/check for this, sorry you got caught